test/parallel_api/dynamic_selection/sycl/test_sycl_sanity.pass.cpp

```diff
@@ -60,24 +60,24 @@ int run_test(sycl::queue q) {
 int test_runner() {
   int r = 0;
   try {
-    sycl::default_selector ds;
-    sycl::queue default_queue(ds);
+    sycl::queue default_queue;
```

Comment:
Can be `auto default_queue = sycl::queue{sycl::defaul_selector_v} `to maintain uniformity but not a big issue.